roles/openshift_grafana/tasks/main.yml

```diff
@@ -0,0 +1,122 @@
+---
+- name: Create grafana namespace
+  oc_project:
+    state: present
+    name: grafana
+
+- name:
+  include_role: tasks/gf-permissions.yml
+  when: '{{ gf_oauth }} == true'
```

Comment:
@mtnbikenc that include role, can be done in more efficient way ?!

Reply:
Add some text to the `name:` directive.  You should use `include_tasks:` instead of `include_role:`.  Don't use curly braces in `when:` conditions.

```yaml
- name: Configure Grafana Permissions
  include_tasks: tasks/gf-permissions.yml
  when: gf_oauth | default(false) | bool == true
```
The `gf_oauth` should probably have a default setting in the role defaults instead of setting the default on all the usage in the following tasks.  It is important to use `| bool` as well because the variable can be interpreted as a string when being set in the inventory.